tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Nice set of plots!</p><p>Comment on the large black bar in the final panel? What state is it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>|1111000001&gt; ! Still a domain wall state, but with two additional spins up. Don't really know why we observe that. But it is actually also something that was observed experimentally if I recall correctly.</p>